src/__tests__/CardContainer.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import 'jest-styled-components';
+import renderer from 'react-test-renderer';
+import '@testing-library/jest-dom/extend-expect';
+import CardDetails from "../components/Container/CardDetails.component";
+
+fdescribe('validate cards works properly', () => {
```

Comment:
Do not filter out test to be ran, run all of them, always. Filtering out is useful while developing or implementing, but not when committing your tests.